assets/stylesheets/banner.scss

```diff
@@ -19,6 +19,16 @@
 			width: 220px;
 			height: 121px;
 		}
+
+		&.is-compact {
```

Comment:
Can you elaborate on why you went with this generic styling versus something Stripe specific? Do you see the potential for many "compact logo" banners where the logo should be `100px`?

Reply:
In concept, there's nothing Stripe specific about `100px`, so I thought there was potential to easily use it for other smaller graphics. If there's another "compact" graphic in the future that can't use this rule, it could be differentiated in the future.

That was my thinking – do you think specifically evoking `stripe` in these rules would be preferable?